### PR TITLE
adds woodmen valley setup

### PR DESCRIPTION
**Pull request by redreceipt**

### CODE REVIEW

File: dags/woodmenvalley-rock-content-item-dag.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+from rock.rock_cover_image import fetch_and_save_cover_image
+from rock.rock_content_item_categories import (
+    fetch_and_save_content_item_categories,
+    attach_content_item_categories,
+)
+from rock.rock_features import fetch_and_save_features
+from rock.rock_deleted_content_items_dag import remove_deleted_content_items
+
+
+start_date = datetime(2021, 8, 12)
```
Comment (vinnyjth) [inline dags/woodmenvalley-rock-content-item-dag.py:19]: check your start date ;)

Comment (redreceipt) [reply dags/woodmenvalley-rock-content-item-dag.py:19]: is this start date not ok?

Comment (vinnyjth) [reply dags/woodmenvalley-rock-content-item-dag.py:19]: A start date in the past will cause the delta dag to start on that date and run for every 30 minute period until it catches up with the present date.

Comment (redreceipt) [reply dags/woodmenvalley-rock-content-item-dag.py:19]: oh, nice. since I don't know when this will be merged, I'll just set "catch up" to false

Comment (redreceipt) [reply dags/woodmenvalley-rock-content-item-dag.py:19]: oh ok

Comment (vinnyjth) [reply dags/woodmenvalley-rock-content-item-dag.py:19]: Catching up a day or two is fine, two months might be frustrating.

**Pull request merged**